docs/deployment/gae_flex_datastore.rst

```diff
@@ -131,6 +131,16 @@ Create the files with content as follows.
 
     This allows you to communicate with Datastore within every request.
 
+#.  ``production.ini``
+
+    Your :app:`Pyramid` application should already contain both a ``development.ini`` and a ``production.ini``.
+    For App Engine to communicate with your application, it will need to be listening on port 8080.
+    Assuming you are using the Waitress WSGI server, modify the ``listen`` variable within the ``server:main`` block.
+
+    .. code-block:: ini
+
+        listen = 0.0.0.0:8080
```

Comment:
I'm not sure about using 0.0.0.0. For production we typically use *. See https://github.com/Pylons/pyramid-cookiecutter-starter/blob/latest/%7B%7Bcookiecutter.repo_name%7D%7D/production.ini#L42

Reply:
Let me try this in Flexible, certainly would be good to keep consistent.

Reply:
Yep, this works. Updated the file